### PR TITLE
New: Manually Edit/Override Album Release

### PR DESCRIPTION
**Pull request by Qstick**
#### Database Migration
YES

#### Description
Change Artist URL to use MBID instead of slug
Only show MetadataProfile Album Types on Artist Detail Page
Add Album Detail Page
Add Album Edit Modal (monitor change, release change)
Add link to Album Edit in Albums table on Artist detail page
Remove Tracks and TrackFiles for removed albums when metadata profile is changed
Remove and Readd Tracks when album release is changed
Add path and relative path as table column options for tracks
Add secondary type as table column option for albums

#### Issues Fixed or Closed by this PR
#109 #129 #128 

*

### CODE REVIEW

File: src/NzbDrone.Core/MetadataSource/SkyHook/SkyHookProxy.cs [hunk 1]
```diff
@@ -118,10 +159,12 @@ public List<Artist> SearchForNewArtist(string title)
 
                 SetCustomProvider();
 
-                var httpRequest = customerRequestBuilder.Create()
+                var httpRequest = _customerRequestBuilder.Create()
                                     .SetSegment("route", "search")
                                     .AddQueryParam("type", "artist")
                                     .AddQueryParam("query", title.ToLower().Trim())
+                                    //.AddQueryParam("images","false") // Should pass these on import search to avoid looking to fanart and wiki
```
Comment (danielunderwood) [inline src/NzbDrone.Core/MetadataSource/SkyHook/SkyHookProxy.cs]: Do we want to go ahead and enable these on metadata or wait a bit?

Comment (Qstick) [reply src/NzbDrone.Core/MetadataSource/SkyHook/SkyHookProxy.cs]: I need to find a way to only pass them on an import and not a regular artist add (we like to have images and overviews for that)

**Comment by Qstick** [comment]
Most of the codacy issues are coming from the add Album service which is not being used yet. I’ll go ahead and pull the code and keep it local until we get to that point. That should keep things clean on the repo.

**Comment by webmosher** [comment]
While this is OK, I would like to suggest that there be a default media release configuration or profile. Why does Lidarr pick the weirdest releases for some albums? I don't want Japanese multi-DVD releases. I only want US CDs.

**Pull request merged**